mods/ra/maps/allies-04/allies04.lua

```diff
@@ -24,28 +24,32 @@ ConvoyDelays =
 {
 	easy = { DateTime.Minutes(4), DateTime.Minutes(5) + DateTime.Seconds(20) },
 	normal = { DateTime.Minutes(2) + DateTime.Seconds(30), DateTime.Minutes(4) },
-	hard = { DateTime.Minutes(1) + DateTime.Seconds(30), DateTime.Minutes(2) + DateTime.Seconds(30) }
+	hard = { DateTime.Minutes(1) + DateTime.Seconds(30), DateTime.Minutes(2) + DateTime.Seconds(30) },
+	tough = { DateTime.Minutes(1), DateTime.Seconds(30) }
```

Comment:
This doesn't look right. It's always going to delay for 30 seconds because in the RandomInteger call `low` is 1 minute and `high` is 30 seconds

Reply:
Updated